www/script/project/helpers.js

```diff
@@ -502,10 +503,13 @@ define(function (require) {
             return ret.join("&");
         },
         getCssClassFromStatus: function (status) {
-            var values = Object.keys(css_classes).map(function (key) {
-                return css_classes[key];
-            });
-            return values[status][1];
+            var values = {};
```

Comment:
The same as above, but you need include `prod/script/main.js` and `prod/script/main.js.map`

Reply:
Done with grunt build prod